src/charts/Chart.ts

```diff
@@ -107,10 +108,18 @@ abstract class Chart {
     // since two different kind of 'strategies' should be allowed (SVG / Canvas)
 
 
-    private streamingIntervalIdentifier: number = null;
+    protected streamingIntervalIdentifier: number = null;
 
     // TODO: Inject with annotations?
-    private visibilityObservable: Observable<any> = GlobalInjector.getRegistered('onVisibilityChange'); 
+    private visibilityObservable: Observable<any> = GlobalInjector.getRegistered('onVisibilityChange');
+
+
+    /**
+     * Stored data when pausing
+     * @protected
+     * @memberof Chart
+     */
+    protected storedData: any[];
```

Comment:
I think for now it is safer to discard the data. We will later think if we really want to keep it, and in that case, how we will implement it.